README.md

```diff
@@ -9,18 +9,72 @@ A Swift mixin to use `UITableViewCells`, `UICollectionViewCells` and `UIViewCont
 [![CircleCI](https://circleci.com/gh/AliSoftware/Reusable/tree/master.svg?style=svg)](https://circleci.com/gh/AliSoftware/Reusable/tree/master)
 [![Platform](http://cocoapod-badges.herokuapp.com/p/Reusable/badge.png)](http://cocoadocs.org/docsets/Reusable)
 [![Version](http://cocoapod-badges.herokuapp.com/v/Reusable/badge.png)](http://cocoadocs.org/docsets/Reusable)
-[![Language: Swift 3 & 4](https://img.shields.io/badge/Swift-3%20%26%204-orange.svg)](https://swift.org)
+[![Language: Swift 3](https://img.shields.io/badge/Swift-3-orange)](https://swift.org)
+[![Language: Swift 4](https://img.shields.io/badge/Swift-4-orange)](https://swift.org)
+[![Language: Swift 5](https://img.shields.io/badge/Swift-5-orange)](https://swift.org)
 
-# Requirements
+# Installation
+
+<details><summary><b>Requirements</b>: which Reusable version to use for each Swift Version?</summary>
 
 |  Swift Version |  Reusable Version  |
 |----------------|--------------------|
 |    2.2 & 2.3   |        2.5.1       |
 |       3.0 (†)  |        3.0.0 +     |
 |       4.0      |        4.0.2 +     |
+|       5.0      |        4.1.0 +     |
 
 _(†) The Reusable 3.0 code also compiles with Swift 4, you'll need 4.0.2+ only if you're using Carthage for integration_
 
+</details>
+
+Reusable can be integrated to your Xcode projects using one of the following options:
+
+<details><summary>Installation instructions for <b>Swift Package Manager (SPM)</b></summary>
+
+[Swift Package Manager](https://swift.org/package-manager/) is Apple's decentralized dependency manager to integrate libraries to your Swift projects. It is now fully integrated with Xcode 11
+
+To integrate Reusable into your project using SPM, specify it in your `Package.swift` file:
+
+```swift
+let package = Package(
+    …
+    dependencies: [
+        .package(url: "https://github.com/AliSoftware/Reusable.git", from: "4.1.0"),
+    ],
+    targets: [
+        .target(name: "YourTarget", dependencies: ["Reusable", …])
+        …
+    ]
+)
+```
```

Comment:
@MortyMerr can you validate that the example is right? (Never actually integrated Reusable on another project via SPM, so another look from someone who already did could be nice)

Reply:
I didn't try it directly with Reusable, but I confirm that the syntax used in the example is right.

Reply:
I didn't try it directly with Reusable, but I confirm that the syntax used in the example is right.